DataLayer/MpptData/MpptUnit.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "I_MpptUnit.h"
+
+class MpptUnit : public I_MpptUnit
+{
+public:
+    MpptUnit();
+    virtual ~MpptUnit();
+
+    /* MpptUnit Gets */
+    unsigned char getMpptStatus() const;
+    unsigned short getArrayVoltage() const;
+    unsigned short getArrayCurrent() const;
+    unsigned short getBatteryVoltage() const;
+    unsigned short getTemperature() const;
+
+    /* MpptUnit Sets */
+    void setMpptStatus(const unsigned char& mpptStatus);
+    void setArrayVoltage(const unsigned short& arrayVoltage);
+    void setArrayCurrent(const unsigned short& arrayCurrent);
+    void setBatteryVoltage(const unsigned short& batteryVoltage);
+    void setTemperature(const unsigned short& temperature);
+
+
+private:
+    unsigned char mpptStatus_;
```

Comment:
In general, it's still the responsibility of the PR opener to fix any CI issues even if it's not related. I can help you look into this though, I imagine it's an easy fix.